### PR TITLE
V1.0/pipelinebuild/106924

### PR DESCRIPTION
**Pull request by rkodev**
Manually triggered build that adds the features backingstore, codefiles containing structs and interfaces and shim `me` path

### CODE REVIEW

**Review by baywet — CHANGES_REQUESTED** [review]
Impacted by this issue https://github.com/microsoft/kiota-abstractions-go/issues/61

File: graph_service_client.go [hunk 1]
```diff
@@ -58,3 +61,13 @@ func (m *GraphBaseServiceClient) GetAdapter() abstractions.RequestAdapter {
...
+
+// Me provides operations to manage the user singleton.
+func (m *GraphBaseServiceClient) Me() *if6ffd1464db2d9c22e351b03e4c00ebd24a5353cd70ffb7f56cfad1c3ceec329.UserItemRequestBuilder {
+	urlTplParams := make(map[string]string)
+	for idx, item := range m.pathParameters {
+		urlTplParams[idx] = item
+	}
+	urlTplParams["user%2Did"] = "me-token-to-replace"
+	return if6ffd1464db2d9c22e351b03e4c00ebd24a5353cd70ffb7f56cfad1c3ceec329.NewUserItemRequestBuilderInternal(urlTplParams, m.requestAdapter)
+}
```
Comment (rkodev) [inline graph_service_client.go:73]: Pciked changes from https://github.com/microsoftgraph/msgraph-sdk-go/issues/347

Comment (baywet) [reply graph_service_client.go:73]: are we planning to merge this PR since it has the UUI serialization bug? shouldn't we regenerate?

Comment (baywet) [reply graph_service_client.go:73]: I meant to type UUID

**Comment by baywet** [comment]
closing since this pull request is still impacted by #412

**Pull request closed without merging**